Update/omot_007.txt

```diff
@@ -3789,13 +3789,13 @@ void main()
 //rそう答えればいい。それはとても、わかりきった模範解答。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そう答えればいい。それはとても、わかりきった模範解答。",
-			NULL, "I answered just like that. It was a completely obvious answer.", Line_Normal);
+			NULL, "I could've answered just like that. It would've be a completely obvious answer.", Line_Normal);
 	ClearMessage();
 
 //rしかし俺は、正直に答える。それが、…彼女自身無意識なんだろうけど、俺のことを信頼してありのままを話してくれた梨花ちゃんの心に届く、一番の言葉だと思ったからだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "しかし俺は、正直に答える。それが、…彼女自身無意識なんだろうけど、俺のことを信頼してありのままを話してくれた梨花ちゃんの心に届く、一番の言葉だと思ったからだ。",
-			NULL, "But I answered honestly. And... even if she didn't seem aware of it herself, I thought that telling Rika-chan to trust me was the best thing I could say to get through to her.", Line_Normal);
+			NULL, "But I answered honestly instead. Because... Rika-chan trusted me and told her true feelings and I thought it was the best thing I could say to get through to her, although she didn't seem to told that consciously.", Line_Normal);
```

Comment:
```suggestion
			NULL, "But instead I answered honestly. Because... even though Rika-chan didn't seem conscious of it herself, she trusted me and shared her true feelings. So I thought it was the best thing I could say to get through to her.", Line_Normal);
```